### DISCUSSION TITLE
Windows 10 anim_to_vtk "Error in Anim Files version "

### DISCUSSION THREAD

**Discussion by badbunny-uk** [OpenRadioss/OpenRadioss#1472] [Q&A] [answered]
Hi,  I'm trying to run Radioss on Windows 10 using the OpenRadiss_win64.exe and win_scripts.zip.
I can run simulations and convert the TH files to csv, but I can't convert the h3d files to vtk.

When I run the converter command, eg:
anim_to_vtk_win64.exe cube_TYPE7.h3d >cube_TYPE7.vtk
...the command runs without returning an error, and creates a vtk file. However it's contents are a single line:
"Error in Anim Files version"

I tried a few of the examples (bumper beam, and ice cube drop) with the same result.
Has anyone run into this or have any idea how to fix this?

Thanks, Pete.

P.S. Has anyone managed to get any of the win_scripts files to work? None of them worked for me (in Win 10). I hacked them to get them working: replacing the forward slashes with backslashes (a typo from Linux conversion?); and commenting out all the pushd/popd stuff near the start which doesn't seem to work as intended. But in the end it seemed easier just writing a simple batch file for each run.

**Comment by MarianBulla** [comment, marked as answer]
Hi badbunny,
the input for the converter are the animation files, created by OpenRadioss (not the h3d file) using:
/ANIM/DT
#   TSTART     TFREQ
0.000000 1.000000
/ANIM/ELEM/VONM
/ANIM/ELEM/EPSP
.
.
.
within the engine file (*_0001.rad).

Regarding the win_script, you need to creat a subfolder in the OpenRadioss folder and lounch it from there:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/50aa1cba-1e45-4b37-aaf1-8e7e94f1e40c)

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by badbunny-uk** [comment]
Hi Marian,

Firstly moving the win_scripts folder under the OpenRadioss folder makes all the scripts work properly without hacking, so thanks very much for that! I can't find where I downloaded the scripts from, not sure if that was an instruction I missed, or if that needs clarifying with them? Do you know where to download them from (I can't find where they came from) - I'd like to add some feedback there about this.

Secondly the h3d was being exported by the analysis so I'd assumed it was the required animation output. Looking in the *_00001.rad input files for all of the example Radioss files I had downloaded from the website, all of the animation outputs had been commented out, so I uncommented them, and it all works now !

For clarity in case someone else is looking at this in the future, in one of the example simulations from the Radioss website 'cube_TYPE7_0001.rad' the file as downloaded has the following lines. The # character at the start of all of these lines comments them out, ie prevents them being used. Just remove the # from all of these lines to enable the animation output:
#/ANIM/VECT/CONT
#/ANIM/VECT/VEL
#/ANIM/BRICK/TENS/STRESS/000       
#/ANIM/ELEM/EPSP
#/ANIM/MASS
#/ANIM/DT
#0.000000000000000    0.001000000000000

Thanks again, Pete.

**Reply by MarianBulla** [reply]
Hi Pete,
First, I'm very happy, it's working on your side.

And thank you very much for your feedback !

Yes, we will improve the documentation page, to avoid confusion for other users.

Thanks and best regards,
Marian
